packages/perseus/src/util/snowman-utils.ts

```diff
@@ -0,0 +1,149 @@
+export enum WidgetType {
```

Comment:
Interesting! I think [Matthew is right](https://github.com/Khan/perseus/pull/1058), we may want to discuss how the heck to manage all these widget lists as they are starting to get [voluminous](https://github.com/Khan/perseus/pull/1058/files). 

They all seem unique, but I wonder if there's something we can do to simplify the amount of work that we would need to do if we add a new widget, or rename an old one. Sync tags could work, but I'm not super stoked on the idea of them as I always found them a little frustrating to deal with.

Reply:
I keep thinking that there must be a way in TypeScript to use a utility type to extract the widget types from the `all-widgets.ts` export (https://github.com/Khan/perseus/blob/9917dceaf750b18122880d9096bddc3b859b9dfa/packages/perseus/src/all-widgets.ts#L5-L6) 

Maybe a question for #typescript ?

Reply:
Thanks again for pointing this out, Third, and for chiming in, Jeremy! I created a ticket for us to come back to this and to consider extracting the types from `all-widgets.ts`